ntoskrnl/ke/amd64/kiinit.c

```diff
@@ -32,7 +32,7 @@ KIPCR KiInitialPcr;
 UCHAR DECLSPEC_ALIGN(16) P0BootStackData[KERNEL_STACK_SIZE] = {0};
 UCHAR DECLSPEC_ALIGN(16) KiDoubleFaultStackData[KERNEL_STACK_SIZE] = {0};
 ULONG_PTR P0BootStack = (ULONG_PTR)&P0BootStackData[KERNEL_STACK_SIZE];
-ULONG_PTR KiDoubleFaultStack = (ULONG_PTR)&KiDoubleFaultStackData[KERNEL_STACK_SIZE];
+ULONG_PTR KiDoubleFaultStack = (ULONG_PTR)&KiDoubleFaultStackData[KERNEL_STACK_SIZE - 8];
```

Comment:
Hmmm...
Why does such shift seems "necessary" there, but not in x86? https://git.reactos.org/?p=reactos.git;a=blob;f=ntoskrnl/ke/i386/kiinit.c;hb=2047cf361327fe1efc46ef58e2ac0e8595d7d468#l19

Reply:
x86 probably doesn't use SSE instructions in the kernel that need 16 byte alignment, so this doesn't apply. It also depends on how the trap entry allocates stack. But after double checking I think this change is actually wrong. I just don't get too many double fault aborts these days to notice 😄 
Double faults push an error code on the stack, so the machine frame is 48 bytes large. The handler allocates 0x160 bytes (same on Windows) of stack, which should leave the stack 16 byte aligned.
I'll revert that part.